guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,51 @@
+[id="installing-fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Installing fapolicyd on {ProjectServer}
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+= Installing fapolicyd on {SmartProxyServer}
+endif::[]
+
+ifeval::["{context}" == "{project-context}"]
+You can install `fapolicyd` along with {ProjectServer} or can be installed on an existing {ProjectServer}.
+If you are installing fapolicyd along with the new {ProjectServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {ProjectServer} rules automatically.
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+You can install `fapolicyd` along with {SmartProxyServer} or can be installed on an existing {SmartProxyServer}.
+If you are installing fapolicyd along with the new {SmartProxyServer}, the installation process will detect the fapolicyd in your {EL} host and deploy the {SmartProxyServer} rules automatically.
+endif::[]
+
+.Prerequisite
+* Ensure your host has access to the BaseOS repositories of {EL}.
+
+.Procedure
+. Install fapolicyd:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+ifndef::foreman-deb[]
+# dnf install fapolicyd
+endif::[]
+ifdef::foreman-deb[]
+# {package-install-project} fapolicyd
+endif::[]
+----
+. Start the `fapolicyd` service:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl enable --now fapolicyd
+----
+
+.Verification
+* Verify that the `fapolicyd` service is running correctly:
++
+[options="nowrap" subs="+quotes"]
+----
+# systemctl status fapolicyd
+----
+
+In case of new {ProjectServer} or {SmartProxyServer} installation, follow the standard installation procedures after installing and enabling fapolicyd on your {EL} host.
```

Comment:
```suggestion
.New {ProjectServer} or {SmartProxyServer} installations
In case of new {ProjectServer} or {SmartProxyServer} installation, follow the standard installation procedures after installing and enabling fapolicyd on your {EL} host.
```

Reply:
@maximiliankolb @adamlazik1 
I and @asteflova think this is a good idea to go with as it is getting confused with the Verification step from the readers perspective. What is your view on this?

Reply:
Sure, I don't object.